desktop/app/window-handlers/notifications/index.js

```diff
@@ -127,6 +130,9 @@ module.exports = function ( mainWindow ) {
 
 				// Tracks API call
 				mainWindow.webContents.send( 'notification-clicked', notification );
+
+				notificationBadgeCount--;
```

Comment:
I think we should probably ensure there's a floor to the count with something like `notificationBadgeCount = Math.max(notificationBadgeCount--, 0)` to guard against the potential of the count dropping below zero.